### PR TITLE
add root user check for mastercontainer

### PR DESCRIPTION
**Pull request by Zoey2936**

### CODE REVIEW

File: Containers/mastercontainer/start.sh [hunk 1]
```diff
@@ -20,6 +20,11 @@ case "${1}" in
 esac
 }
 
+if [ "$(whoami)" != "root" ] || [ "$(id -u)" != "0" ] || [ "$(id -g)" != "0" ]; then
+    print_red "Container does not run as root. Cannot continue."
```
Comment (szaimen) [inline Containers/mastercontainer/start.sh:24]:
```suggestion
# Check if running as root user
if [ "$EUID" != "0" ]; then
    print_red "Container does not run as root user. This is not supported."
```

Comment (Zoey2936) [reply Containers/mastercontainer/start.sh]: I've added this changed and I didn't see the change of the if condition, is EUID even a valid env?

Comment (szaimen) [reply Containers/mastercontainer/start.sh]: Yes, it is. Just try it :)

Comment (Zoey2936) [reply Containers/mastercontainer/start.sh]: what is there the difference to id -u / id -g / whoami?

Comment (szaimen) [reply Containers/mastercontainer/start.sh]: Both would work but using the variable seems cleaner to me :)

Comment (Zoey2936) [reply Containers/mastercontainer/start.sh]: but what if the group is different?

Comment (szaimen) [reply Containers/mastercontainer/start.sh]: I think that doesnt matter since you have root permissions anyway?

Comment (Zoey2936) [reply Containers/mastercontainer/start.sh]: not sure...

**Review by szaimen — APPROVED** [review]

**Pull request merged**